### PR TITLE
Fix #1819: Crash on reconnect 1.5.8-release-20704.0.000

### LINKED ISSUE #1819: Crash on reconnect 1.5.8-release-20704.0.000
1. Connect to a server
1. Reconnect in console
3. Crash

```
Version = 1.5.8-release-20704.0.000
Time = Tue Nov 10 13:49:11 2020
Module = F:\steamapps\steamapps\common\Grand Theft Auto San Andreas\gta_sa.exe
Code = 0xC0000005
Offset = 0x00009CFC

EAX=0000003C  EBX=00000000  ECX=00000000  EDX=0001A602  ESI=00000003
EDI=00000003  EBP=0177F918  ESP=0177F8EC  EIP=00409CFC  FLG=00210283
CS=0023   DS=002B  SS=002B  ES=002B   FS=0053  GS=002B
```

Did not test other versions.

### PR DESCRIPTION
**Pull request by TheNormalnij**
Fix for #1819
Crash reason: MTA try unload some ID's twice

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
I don't think we should merge this PR. This is related to #1840. @TheNormalnij As desribed in a comment [here](https://github.com/multitheftauto/mtasa-blue/pull/1842#discussion_r524168066), please do the same for peds and vehicles.

**Comment by TheNormalnij** [comment]
This is another crash, related to some ID's

File: Client/mods/deathmatch/logic/CClientModelManager.cpp [hunk 1]
```diff
@@ -60,7 +60,7 @@ int CClientModelManager::GetFirstFreeModelID(void)
     for (int i = 0; i < MAX_MODEL_ID; i++)
     {
         CModelInfo* pModelInfo = g_pGame->GetModelInfo(i, true);
-        if (!pModelInfo->IsValid())
+        if (!pModelInfo->IsValid() && (i < 300 || i > 319) )
```
Comment (qaisjp) [inline Client/mods/deathmatch/logic/CClientModelManager.cpp:64]: if this is going to be merged pls can we include a code comment that explains what this is protecting against? someone who reads this in the future might get confused as to what this is guarding

Comment (TheNormalnij) [reply Client/mods/deathmatch/logic/CClientModelManager.cpp:64]: Added

**Pull request merged**